.github/workflows/gh-workflow-approve.yaml

```diff
@@ -0,0 +1,49 @@
+name: Approve Workflow Runs
+
+permissions:
+  actions: write
+  contents: read
+
+on:
+  pull_request_target:
+    types:
+      - labeled
+      - synchronize
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.event.number }}
+  cancel-in-progress: true
+
+jobs:
+  ok-to-test:
```

Comment:
We've been using this action for Trainer, Katib, Model Registry, and Pipelines. to allow trigger actions with these labels:
```
/ok-to-test
/rerun-all
```
https://github.com/kubeflow/trainer/blob/master/.github/workflows/github-trigger-rerun-test.yaml#L13

Do you want to re-use it for Dashboard and Notebooks @thesuperzapper ? 

cc @kubeflow/wg-pipeline-leads @kubeflow/wg-data-leads

Reply:
@andreyvelich after https://github.com/GoogleCloudPlatform/oss-test-infra/pull/2404, the prow bot will handle re-triggering failed GitHub actions when you use `/retest`.

So for the `/ok-to-test` it's probably easier to just use the script-based one, so we don't need to depend on `estroz/rerun-actions` and open ourselves to more supply chain attacks than needed.

If we start having problems, we can discuss other options, but we are successfully using the script on `kubeflow/notebooks`.